modules/tensor_mechanics/doc/content/syntax/Modules/TensorMechanics/Master/index.md

```diff
@@ -25,6 +25,7 @@ The Tensor Mechanics `Master` Action is used to construct the kernels, displacem
 | | [ComputeSmallStrain](/ComputeSmallStrain.md) or [ComputePlaneSmallStrain](/ComputePlaneSmallStrain.md) or [ComputeAxisymmetric1DSmallStrain](/ComputeAxisymmetric1DSmallStrain.md) or [ComputeAxisymmetricRZSmallStrain](/ComputeAxisymmetricRZSmallStrain.md) | |
 | | [ComputeIncrementalSmallStrain](/ComputeIncrementalSmallStrain.md) or [ComputePlaneIncrementalStrain](/ComputePlaneIncrementalStrain.md) or [ComputeAxisymmetric1DIncrementalStrain](/ComputeAxisymmetric1DIncrementalStrain.md) or [ComputeAxisymmetricRZIncrementalStrain](/ComputeAxisymmetricRZIncrementalStrain.md) |`incremental` : boolean for using a incremental strain formulation |
 | Add AuxVariables and AuxKernels for various tensor component and quantity outputs | [AuxVariables](/AuxVariables/index.md) and [RankTwoAux](/RankTwoAux.md) or [RankTwoScalarAux](/RankTwoScalarAux.md) or [RankFourAux](/RankFourAux.md) | `generate_output`: a string of the quantities to add |
+| Couple the optional [GlobalStrain](/GlobalStrain/index.md) calculation | None, [GlobalStrain](/GlobalStrainAction.md) system is set up separately| `global_strain`: name of the material property that computes the global strain tensor |
```

Comment:
Sorry, I wasn't clear about which column, how about:

```
| Add a global strain tensor for visualizing periodic simulations (optional)  | Couples to the [GlobalStrain](/GlobalStrainAction.md) system | `global_strain`: name of the material property that computes the global strain tensor |
```

I'd like to avoid the `None` in the second column if possible. I'm also not sure about the functionality of the global strain, so please correct that description in the first column